pages/fadderperioden.tsx

```diff
@@ -11,7 +11,7 @@ import { groq } from "next-sanity";
 import { TypedObject } from "sanity";
 import { FPGroups } from "@/schemas/dayDescription";
 import { FACEBOOK_GROUP_FIRSTYEARS, MTDT, MTKOM } from "@/utils/constants";
-import { sanityClient } from "@/utils/sanity";
+import { sanityClient, sanityFetch } from "@/utils/sanity";
```

Comment:
```suggestion
import { sanityFetch } from "@/utils/sanity";
```
Looks like the sanityClient import is not used anymore in this file, maybe not in the others either